plugins/official/index.ts

```diff
@@ -1 +1,3 @@
 export { stackSnippetPlugin } from "./stack-snippets/src/stackSnippetPlugin";
+export { StackSnippetOptions } from "./stack-snippets/src/common";
+export { SnippetMetadata } from "./stack-snippets/src/common";
```

Comment:
NIT: make it explicit we are exposing types
```suggestion
export type { StackSnippetOptions, SnippetMetadata } from "./stack-snippets/src/common";
```